### ISSUE TITLE
liberate documentation

### ISSUE THREAD

**Issue by kayiwa** [pulibrary/ops-catchall#98]
Can all our documentation be under distributed version control systems (pronounced). Documentation for the Ops team lives on Github, Sharepoint, people's laptop. There's a tension of revealing too much on Github. Will this be a [comfortable medium](https://forgejo.org/docs/v1.19/user/authentication/)

**Comment by acozine** [comment]
Yes! After discussion at our 2024 Ops Team Retreat we explored options for consolidating documentation. @Beck-Davis created a [spreadsheet](https://docs.google.com/spreadsheets/d/1dr_415Z33S4ZW3KogznHUUMLJe5I4-9fm47LIf4hxDg/edit?gid=630110278#gid=630110278) of our current documentation and @kayiwa created a locally-hosted [GitLab instance](https://gitlab.lib.princeton.edu/) to host version-controlled documentation.

See also the notes in the 23 January entry in our [Virtual Infrastructure Updates doc for 2025](https://docs.google.com/document/d/1SwL8SG56K5gKRhgRVnWQG3iIv6dq_gDnKDEd2T9XwtE/edit?tab=t.0#heading=h.f8yzzuq1oq45).
